x-pack/legacy/plugins/apm/public/components/app/Settings/CustomizeUI/CustomLink/CustomLinkTable.tsx

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiFieldSearch,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiSpacer
+} from '@elastic/eui';
+import { isEmpty } from 'lodash';
+import { units, px } from '../../../../../style/variables';
+import { CustomLink } from '../../../../../../../../../plugins/apm/server/lib/settings/custom_link/custom_link_types';
+import { ManagedTable } from '../../../../shared/ManagedTable';
+import { TimestampTooltip } from '../../../../shared/TimestampTooltip';
+import { LoadingStatePrompt } from '../../../../shared/LoadingStatePrompt';
+
+interface Props {
+  items: CustomLink[];
+  onCustomLinkSelected: (customLink: CustomLink) => void;
+}
+
+export const CustomLinkTable = ({
+  items = [],
+  onCustomLinkSelected
+}: Props) => {
+  const [searchTerm, setSearchTerm] = useState('');
+
+  const columns = [
+    {
+      field: 'label',
+      name: i18n.translate(
+        'xpack.apm.settings.customizeUI.customLink.table.name',
+        { defaultMessage: 'Name' }
+      ),
+      truncateText: true
+    },
+    {
+      field: 'url',
+      name: i18n.translate(
+        'xpack.apm.settings.customizeUI.customLink.table.url',
+        { defaultMessage: 'URL' }
+      ),
+      truncateText: true
+    },
+    {
+      width: px(160),
+      align: 'right',
+      field: '@timestamp',
+      name: i18n.translate(
+        'xpack.apm.settings.customizeUI.customLink.table.lastUpdated',
+        { defaultMessage: 'Last updated' }
+      ),
+      sortable: true,
+      render: (value: number) => (
+        <TimestampTooltip time={value} timeUnit="minutes" />
+      )
+    },
+    {
+      width: px(units.quadruple),
+      name: i18n.translate(
+        'xpack.apm.settings.customizeUI.customLink.table.links',
+        { defaultMessage: 'Links' }
```

Comment:
We shouldn't need to show a table row heading for the actions. E.g. from configurations

<img width="386" alt="Screenshot 2020-03-05 at 14 43 58" src="https://user-images.githubusercontent.com/4104278/75987176-c398ff00-5eef-11ea-8052-fcd8431ee696.png">